packages/dicom/dcmtk/CMakeLists.txt

```diff
@@ -1,10 +1,40 @@
 find_package(ITK REQUIRED
   COMPONENTS
     ITKDCMTK
+    ITKLabelMap
+    ITKThresholding
     WebAssemblyInterface
   )
 include(${ITK_USE_FILE})
 
+if(POLICY CMP0077)
+  set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+endif()
+
+include(FetchContent)
+# DCMQI
+set(DCMQI_GIT_REPOSITORY "https://github.com/jadh4v/dcmqi.git")
```

Comment:
update to upstream if possible

Reply:
Will update in a separate PR after dcmqi branch is merged, see here:
https://github.com/QIICR/dcmqi/pull/507